wp-seo-main.php

```diff
@@ -315,7 +315,7 @@ function wpseo_init() {
 		$GLOBALS['wpseo_sitemaps'] = new WPSEO_Sitemaps();
 	}
 
-	if ( ! defined( 'DOING_AJAX' ) || ! DOING_AJAX ) {
+	if ( ! wp_doing_ajax() ) {
```

Comment:
Worth noting that when using `!` the behavior is slightly different because previously it was an OR and now it's an AND. For example, try to add the filter `add_filter( 'wp_doing_ajax', '__return_true' );` before this condition and see the different behavior with the previous and new syntax (e.g. observe the toolbar menu).
Seems to me the new behavior is more correct, I'd like this to be double checked by @andizer though :)

Reply:
Thats correct 👍